clueless/views.py

```diff
@@ -179,21 +179,35 @@ def playgame(request, game_id):
 	return HttpResponse(template.render(context,request))
 
 @login_required
+
 def playerturn(request, game_id):
+	context = {}
 	template = loader.get_template('clueless/playerturn.html')
 
-	game = Game.objects.get(id=game_id)
-	context = {}
+	#get request variables
+	user_id = request.user
+	#mocked for now, will get for real in jerrold's branch
+	game = Game.objects.get(id = game_id)
 	context['game'] = game
+	player = Player.objects.get(user = user_id, currentGame=game)
+
+	if request.method == 'GET':
```

Comment:
also good call! had that redirection written down as a bug but hadn't looked into it yet so good to know this is the issue, thank you!